resources/php_ini.rb

```diff
@@ -0,0 +1,26 @@
+resource_name :php_ini
+
+default_action :add
+
+property :mode, String, default: '0644'
+property :options, Hash, default: {}, required: true
+
+action :add do
+
+  directory '/etc/php.d' do
+    mode '0755'
+  end
```

Comment:
Instead of creating the directory, I recommend you include the ``osl-php::default`` recipe here.

Reply:
I think including `osl-php::default` here may be problematic as-is. This resource is going to be used in `osl-php::opcache` and `osl-php::apc`. The way I was setting things up with the other PR in this repository, is the default recipe would include opcache if `node['osl-php']['use_opcache']` is true. I did it that way so you'd only have to modify a few attributes and have it set up. 
If we make the resource include `osl-php::default` it wouldn't ever do anything with opcache since the default recipe is already included once. Probably more a hint that I need to change how the opcache recipe works/is invoked

Reply:
I guess that's OK however I don't think you need to set the mode.